src/coreclr/System.Private.CoreLib/src/System/Reflection/RuntimeParameterInfo.cs

```diff
@@ -453,6 +466,8 @@ private static DateTime GetRawDateTimeConstant(CustomAttributeData attr)
 
         private static object? GetRawConstant(CustomAttributeData attr)
         {
+            // FIXME: Why are we relying only on named arguments, constructor arguments
```

Comment:
I guess it is what the historic behavior has always been. We should make sure to have test for it.